docs/api/internal.json

```diff
@@ -256,6 +256,11 @@
             "example": "9788656",
             "description": "The unique identifier for the job in the site (post ID)."
           },
+          "job_status": {
```

Comment:
Risking to be repetitive here but: what about just status? All the other fields (title, etc) are "job title". So I think the status field is no different?

If you chose job_status for some special reason feel free ignore this comment :)

Reply:
Makes sense! Sorry that I misunderstood it previously. https://github.com/Automattic/WP-Job-Manager/pull/2544/commits/b1e2e52b352811320d77a97a91550f087dc0a0f6